server/src/main/scala/drt/server/feeds/lhr/forecast/LHRForecastEmail.scala

```diff
@@ -0,0 +1,93 @@
+
+package drt.server.feeds.lhr.forecast
+
+import java.io._
+import javax.mail._
+import javax.mail.internet.InternetAddress
+import javax.mail.search.FromTerm
+
+import org.slf4j.LoggerFactory
+
+import scala.util.{Failure, Success, Try}
+
+object LHRForecastEmail {
+
+  val log = LoggerFactory.getLogger(getClass)
+
+  def apply(mailHost: String,
```

Comment:
this is a bit weird - all the defs in the apply def

convert to case class with a getLatestAttachment()?